src/client/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "lodash": "^4.17.10",
+    "node-fetch": "^2.1.2",
+    "prop-types": "^15.6.1",
+    "react": "^16.3.2",
+    "react-dom": "^16.3.2",
+    "react-router": "^4.2.0",
+    "react-router-dom": "^4.2.2",
+    "react-scripts": "1.1.4",
+    "socket.io-client": "^2.1.0"
+  },
+  "proxy": "http://localhost:5000",
```

Comment:
What's that proxy is for? React web server, or frontend?

Reply:
It proxies all frontend requests through the server

Reply:
What if frontend is not opened on the same computer, as web server?

Reply:
Once we have frontend merge I'll be building the current master's frontend to production build files so that the webserver can serve static files as it should. The proxy is purely for development purposes. But in this case, it would not function if you hosted the server somewhere other than the localhost.

Reply:
Make a ticket for that, with priority label

Reply:
#61